src/test/fuzz/coins_view.cpp

```diff
@@ -87,7 +88,9 @@ void TestCoinsView(FuzzedDataProvider& fuzzed_data_provider, CCoinsView& backend
                 (void)coins_view_cache.Sync();
             },
             [&] {
-                coins_view_cache.SetBestBlock(ConsumeUInt256(fuzzed_data_provider));
+                uint256 best_block{coins_view_cache.GetBestBlock()};
+                if (is_db && best_block.IsNull()) best_block = uint256::ONE;
+                coins_view_cache.SetBestBlock(best_block);
```

Comment:
To confirm: the removal of `ConsumeUint256()` here is okay (and _doesn't_ cause this test to never rotate the best block) because of the `BatchWrite()` call below, right? 

For `!is_db` cases this is now a no-op, whereas before it wasn't. Is that intended?

Reply:
It's been a while since i wrote this code and to be honest i don't remember what i was thinking. (Note-to-self: comments are good actually.)

As you point out it does change the behaviour of the existing target. Since i don't have a rationale i've updated it to simply keep the existing behaviour.